src/components/dialogs/contingency-list-selector.jsx

```diff
@@ -146,30 +141,25 @@ const ContingencyListSelector = (props) => {
     };
 
     const removeFromFavorite = (toRemove) => {
-        const toDelete = new Set(toRemove);
```

Comment:
The Set performs existence checks more efficiently than Array.includes, i think this is the purpose of doing this

Reply:
Those are really litlle arrays, I wanted to keep it simple but ths is right. will revert

Reply:
fixed